management/src/main/scala/akka/management/AkkaManagementSettings.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.management
+
+import java.net.InetAddress
+
+import com.typesafe.config.Config
+
+import scala.collection.JavaConverters._
+import scala.collection.immutable
+
+final class AkkaManagementSettings(val config: Config) {
+  private val managementConfig = config.getConfig("akka.management")
+
+  object Http {
+    private val cc = managementConfig.getConfig("http")
+
+    val Hostname: String = {
+      val hostname = cc.getString("hostname")
+      if (hostname == "") InetAddress.getLocalHost.getHostAddress
```

Comment:
Use the style we do in artery instead with `<getHostAddress>` or something like that to trigger this?

Reply:
ok